File: Ch2/02_01/fizzbuzz.py

#average of all numbers between 1 to 100 which is divided by either 3 or 5.
count,total=0,0
for i in range(1,101):
    if i%3==0 or i%5==0:
        count=count+1
        total=total+i
average=total/count
print(average)

File: Ch2/02_01/fizzbuzz.go
package main

import "fmt"

func main() {
	count, total := 0, 0
	for i := 1; i <= 100; i++ {
		if (i%3 == 0) || (i%5 == 0) {
			count++
			total = total + i
		}
	}
	average := float64(total) / float64(count)
	fmt.Println(average)
	fmt.Println(float64(total) / float64(count))
}
